lib/orglogin.dart

```diff
@@ -0,0 +1,258 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/src/foundation/key.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class Orglogin extends StatefulWidget {
+  const Orglogin({Key? key}) : super(key: key);
+
+  @override
+  State<Orglogin> createState() => _OrgloginState();
+}
+
+class _OrgloginState extends State<Orglogin> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      extendBodyBehindAppBar: true,
+      appBar: AppBar(
+        title: const Text(
+          'Profile Setup',
+          //   style: TextStyle(
+          //       fontWeight: FontWeight.bold, color: Colors.black, height: 24),
+        ),
+        centerTitle: true,
+        titleTextStyle: TextStyle(
+          color: Colors.black,
+        ),
+        elevation: 0,
+        backgroundColor: Color.fromARGB(0, 5, 0, 0),
+      ),
+      body: Stack(
+        children: [
+          Container(
+            decoration: const BoxDecoration(
```

Comment:
Just keep the decoration of the container. This will serve as the background in the Stack. Then add one other layer only, i.e a Container that will act as the foreground.
So, the layout should be: 
       Stack(
           children: [
               Container(), // decoration background
               Container(), // to contain all other foreground items
           ]
       )